data-prepper-plugins/elasticsearch/README.md

```diff
@@ -42,47 +42,18 @@ pipeline:
 
 The elasticsearch sink will reserve `otel-v1-apm-service-map` as index for record ingestion.
 
-### Custom data
-
-```$xslt
-pipeline:
-  ...
-  sink:
-    elasticsearch:
-      hosts: ["https://localhost:9200"]
-      username: YOUR_USERNAME_HERE
-      password: YOUR_PASSWORD_HERE
-      index: "some-index"
-      template_file: /your/local/template-file.json
-      document_id_field: "someId"
-      dlq_file: /your/local/dlq-file
-      bulk_size: 4
-```
-
-User needs to provide custom index for record ingestion.
-
-## Parameters
-
-### Hosts
+## Configuration
 
-A list of IP addresses of elasticsearch nodes.
+- `hosts`: A list of IP addresses of elasticsearch nodes.
 
-### cert
-CA certificate that is pem encoded. Accepts both .pem or .crt. This enables the client to trust the CA that has signed the certificate that ODFE is using.
+- `cert`(optional): CA certificate that is pem encoded. Accepts both .pem or .crt. This enables the client to trust the CA that has signed the certificate that ODFE is using.
 Default is null. 
 
-### Username
+- `username`(optional): A String of username used in the [cognito](https://opendistro.github.io/for-elasticsearch-docs/docs/security/access-control/users-roles) of ODFE cluster. Default is null.
 
-A String of username used in the [cognito](https://opendistro.github.io/for-elasticsearch-docs/docs/security/access-control/users-roles) of ODFE cluster.
-
-### Password
-
-A String of password used in the [cognito](https://opendistro.github.io/for-elasticsearch-docs/docs/security/access-control/users-roles) of ODFE cluster.
-
-### Trace analytics raw
-
-A boolean flag indicates APM trace analytics raw span data type. e.g.
+- `password`(optional): A String of password used in the [cognito](https://opendistro.github.io/for-elasticsearch-docs/docs/security/access-control/users-roles) of ODFE cluster. Default is null.
```

Comment:
What is "cognito" in this context? If I click that link I don't see a reference to AWS Cognito, so I'm assuming we're talking about ODFE's built-in users/roles feature.

Reply:
The link is for ODFE. But it should also work for AES if it is configured with user credential cognito. I will add AES cluster and probably provide a similar link if I can find one.

Reply:
Will push back the change as mentioned in slack that AES setting needs to be tested and finalized

Reply:
My comment was more targeted at the term "cognito". [Amazon Cognito](https://aws.amazon.com/cognito/) is an AWS offering, so not really sure what "cognito" is in terms of ODFE. Maybe replace with more appropriate ODFE/FGAC terminology?

Reply:
I will replace it with `internal user`.